packages/react/src/themes/teams/components/Dropdown/dropdownSearchInputStyles.ts

```diff
@@ -6,6 +6,8 @@ const dropdownSearchInputStyles: ComponentSlotStylesInput<
   DropdownSearchInputProps,
   DropdownVariables
 > = {
+  root: {},
```

Comment:
What was the motivation for this? As I see in `DropdownSearchInput.tsx` we are not using styles.root, nor classes.root. I don't understand why we need this?